string way
int64 speed
float64 persent
